save-backend/src/main/kotlin/org/cqfn/save/backend/repository/ProjectRepository.kt

```diff
@@ -0,0 +1,8 @@
+package org.cqfn.save.backend.repository
+
+import org.cqfn.save.backend.entities.Project
+import org.springframework.context.annotation.Profile
+import org.springframework.data.jpa.repository.JpaRepository
+
+@Profile("dev")
```

Comment:
Profile shouldn't be needed here, repositories should be available in all profiles

Reply:
Removed